inst/WORDLIST

```diff
@@ -1,134 +1,132 @@
-Aga
 AKU
-al
```

Comment:
I automatically configured these changes with `usethis::use_spell_check()`, which re-ordered the `WORDLIST` file (capitalized strings first, then lowercase-initial strings). The re-ordering shouldn't have any consequences.